e2e-tests/.ci/.e2erc

```diff
@@ -0,0 +1,72 @@
+# Utility variables
+# NB: these assume you `source` them from the directory this file is in
+export MME2E_DC_SERVER="docker-compose -p mmserver -f $(readlink -e ../../server/build/gitlab-dc.postgres.yml) -f $(readlink -e ./server.override.yml)"
```

Comment:
I wonder if we can drop the prefix `MM` or `mm` to make more readable unless it's made not to conflict with other.

Reply:
I changed the way the variables work, and dropped the `MME2E_` prefix from all user facing variables, and hopefully made much clearer in the readme what their flow is (so e.g. one does not expect to run `MYVAR=myvalue make` and expect the `MYVAR` variable to be accessible by the server or cypress).

The prefix is still there for the internal variables though, which I think helps distinguish those from the user-facing ones. Shall I keep it this way, or update those variables as well?

Reply:
Sounds good to me, thanks!